flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/FlinkCatalog.java

```diff
@@ -190,6 +191,12 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
             throw new UnsupportedOperationException(
                     "Only support CatalogTable, but is: " + table.getClass());
         }
+        if (tablePath.getObjectName().contains(METADATA_TABLE_SPLITTER)) {
```

Comment:
Maybe this validation can be in `AbstractCatalog.getTableLocation`?

Reply:
Nice, I think `AbstractCatalog.getTableLocation` is better

Reply:
Done